index.html

```diff
@@ -2,47 +2,61 @@
 <html lang='en'>
 
 <head>
-  <meta charset='utf8'>
-  <title>Arraying</title>
+    <meta charset='utf8'>
```

Comment:
you should change the _charset='utf8'_ to _charset='utf-8'_ because if you validate the html later then you will get error.

Reply:
Thank you